src/app/components/apps-view/apps-view.component.ts

```diff
@@ -301,4 +303,22 @@ export class AppsViewComponent implements OnInit {
       this.removeRowSelection();
     }
   }
+
+  formatResources(colValue:string):string[]{
+    let result:string[]=colValue.split("<br/>")
+    return result;
+  }
+
+  getToggle(cellId: string):boolean{
+    return this.cellToggle.get(cellId)!
+  }
+
+  toggle(cellId: string){
+    this.cellToggle.set(cellId, !this.getToggle(cellId))
+  }
+
+  getCellId(element: any, colId: any):string{
+    let cellId =`${element.applicationId}-${colId}`
+    return cellId
+  }
```

Comment:
Seems like `result` and `cellId` aren't modified, so we can use the `const` instead of `let`;
And also seems like missing some semicolons.

```js
 formatResources(colValue: string): string[] {
    const result: string[] = colValue.split("<br/>");
    return result;
  }

  getToggle(cellId: string): boolean {
    return this.cellToggle.get(cellId)!
  }

  toggle(cellId: string): void {
    this.cellToggle.set(cellId, !this.getToggle(cellId));
  }

  getCellId(element: any, colId: any): string {
    const cellId =`${element.applicationId}-${colId}`;
    return cellId;
  }
```